source/_docs/domains.md

```diff
@@ -161,3 +160,25 @@ All redirect logic should include the `php_sapi_name() != "cli"` conditional sta
 Errors referencing too many redirects may be a result of using the ` $_SERVER['HTTP_X_FORWARDED_PROTO']` variable within redirect logic located in your site's `wp-config.php` or `settings.php` file. Resolve this error by replacing the offending redirect logic with the [recommended code samples in the above section](/docs/domains/#redirect-to-https-and-the-primary-domain) and for your specific use case.
 #### Mixed-mode browser warnings
 Replace `http://` in the site's database and configure your CMS to assume users are visiting via HTTPS and the site’s primary domain. Templates for example should reference HTTPS in absolute CSS and Javascript sources, even when accessed with HTTP.
+
+
+## Troubleshooting
+
+### Add A/AAAA records if adding CNAME not possible
+
+If your domain has a MX or TXT record and prevents the use of a CNAME, as recommended by the site dashboard, add A and AAAA records. Find the required DNS values  by querying the environment's platform domain. Make sure to replace `example` with your actual platform domain.
+
+*Use a free online tool like WhatsmyDNS.net:*
+
+A record: https://www.whatsmydns.net/#A/live-example.pantheonsite.io 
```

Comment:
Would it be better to include invalid DNS characters in the example to draw attention to the portion that needs to be replaced?  e.g. `{example}` instead of `example`?